assets/dev/js/editor/components/settings/base/manager.js

```diff
@@ -213,13 +213,13 @@ module.exports = elementorModules.ViewModule.extend( {
 		self.debounceSave();
 	},
 
-	onElementorDocumentLoaded: function() {
+	onElementorDocumentLoaded: function( document ) {
 		this.updateStylesheet();
 
 		this.addPanelPage();
 
 		if ( ! elementor.userCan( 'design' ) ) {
-			$e.route( 'panel/page-settings/settings' );
+			document.config.panel.default_route = 'panel/page-settings/settings';
```

Comment:
Move it the the server side. the JS shouldn't touch the server config.

Reply:
Handled.